src/windows/common/tools/README.md

```diff
@@ -0,0 +1,5 @@
+#### Tools included in this folder:
+
+KDbgCtrl (Kernel Debugging Control) -- Used to change the dump configuration without rebooting the system. For more info, see https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/using-kdbgctrl
+
+Filename -- kdbgctrl.exe -- SHA 256 Hash --  A36BFEC3572661B5B8C04663DB87DD5B1BC39F16086869DE0DFD568D9EB71D7C
```

Comment:
Lets try to add file version and architecture and a few other info on the file

File Name: kbdgctrl.exe
Created By: Microsoft Corporation
Target Architecture: x64
File Version: 10.0.20279.1
File Size: 76.2KB
SHA 256 Hash: A36BFEC3572661B5B8C04663DB87DD5B1BC39F16086869DE0DFD568D9EB71D7C

Reply:
Also maybe a link to https://docs.microsoft.com/en-us/windows-insider/flight-hub/ to check for the different version of the tool if needed